feature-libs/asm/components/customer-emulation/customer-emulation.component.html

```diff
@@ -4,6 +4,17 @@
     <label class="cx-asm-uid">{{ customer?.uid }}</label>
   </div>
   <cx-asm-bind-cart></cx-asm-bind-cart>
+  <!-- TODO(CXSPA-3090): Remove feature flag in 7.0 -->
```

Comment:
[MEDIUM REMARK]
let's remove this html comment. As far as I understand, otherwise it may land in customer's production applications.